### ISSUE TITLE
Question about the Ligands in Peptides Holdout and 0 Ligand Bias Sets

### ISSUE THREAD

**Issue by ChungMinjae** [guydurant/toolboxsf#3]
Hello,

Thank you for the excellent paper and the ToolBoxSF repository.

I have a quick question to confirm my understanding of the benchmark construction:

For the Peptides Holdout set, my understanding is that it tests generalization to a new chemical class (peptides).

For the 0 Ligand Bias set, my understanding is that the specific ligand molecules (by InChI-Key) in the test set are completely held out from the training set.

To summarize: is it correct to say that for both benchmarks, the ligands in the test sets are completely absent from the training set?

Thanks for any clarification you can provide!

**Comment by guydurant** [comment]
Yes, that is correct, the ligands in the test-sets are not present in their respective training data. So each benchmark requires retraining for its accompanying training set.
